nsis/src/nsis/format/NsisScriptHeader.java

```diff
@@ -17,71 +17,80 @@
 	private int archiveSize;
 	private int compressedHeaderSize;
 	private int flags;
+	private static Structure STRUCTURE;
```

Comment:
Only constant (final) fields should use UPPERCASE for naming per style guide: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
structure is now final